### PR TITLE
Migrated use-database.js to TypeScript

### PR DESCRIPTION
**Pull request by Abdulrehman-Jafer**

### CODE REVIEW

**Comment by Abdulrehman-Jafer** [comment]
Forgot to staged the deleted .js file in first commit

File: apps/web/src/hooks/use-database.ts [hunk 1]
```diff
@@ -28,14 +28,14 @@ if (process.env.NODE_ENV === "production") {
...
-export default function useDatabase(persistence) {
+export default function useDatabase(persistence?: "db" | "memory") {
   const [isAppLoaded, setIsAppLoaded] = useState(memory.isAppLoaded);
 
   useEffect(() => {
     if (memory.isAppLoaded) return;
 
     (async () => {
-      await import("../app.css");
+      await import("../app.css" as string);
```
Comment (thecodrr) [inline apps/web/src/hooks/use-database.ts:38]: Why is this needed?

Comment (Abdulrehman-Jafer) [reply apps/web/src/hooks/use-database.ts:38]: This problem I faced and found out Typescript won't let us import the css file it will only allowed .ts file that is why as string is necessary to import the .css file.

Comment (Abdulrehman-Jafer) [reply apps/web/src/hooks/use-database.ts:38]: Otherwise there was an error module not found.

**Review by thecodrr — APPROVED** [review]
